### ISSUE TITLE
Dropdownlist NVDA screen reader reads the list items two times

### ISSUE THREAD

**Issue by PavanJagini** [OfficeDev/office-ui-fabric-react#9887]
## Actual behavior:
NVDA announces list items name twice.
For Ex: For list item "Afghanistan" NVDA announces as "Afghanistan  Afghanistan 1 of 242".

## Expected behavior:
NVDA should announce list item names only once.
For Ex: For list item "Afghanistan" NVDA should announce like a "Afghanistan 1 of 242".
Code snippet:
<Dropdown
                    placeholder={UUMCONSTANTS.USER_DETAILS.ORGANIZATION_LOCALE}
                    name="orglocaleDropdown"
                    disabled={this.props.disabled}
                    options={this.props.allAvailableLocales.map(locale => ({ ...locale, key: locale.Id, text: locale.Name }))}
                    autoFocus
                    selectedKey={this.props.selectedOrgdetails["Locale"] === undefined ? null : this.props.selectedOrgdetails.Locale.key}
                    onChange={(e, value) => {
                        this.props.onLocaleChanged(value.key)
                        this.props.OrgdetailsChange('Locale', value);
                        this.forceUpdate();
                    }}
                    ariaLabel="Select Organization Locale"
                    id="ddlorglocale" />

**Comment by micahgodbolt** [comment]
@PavanJagini did we chat about this via email? I believe that plan was to log this as a bug with NVDA, correct? Dropdown works properly with Narrator/Edge, so these other screen readers with issues must not be to the same spec.

**Comment by PavanJagini** [comment]
Hey, Thanks for your immediate reply. This is the first time, I am raising this bug with you related to this. So, I will take this as a reference to move fwd. Thanks again.
